app/src/pages/Editor/Cursor.tsx

```diff
@@ -10,9 +10,10 @@ import {
 } from '../../core/document';
 import { Pane, useTheme } from 'evergreen-ui';
 import { useElementSize } from '../../components/useElementSize';
+import { Theme } from '../../components/theme';
 
 export function Cursor(): JSX.Element {
-  const theme = useTheme();
+  const theme: Theme = useTheme();
```

Comment:
If you're finding yourself doing this frequently throughout your app, it might be worth it to break out a wrapper `useTheme` hook that is always default typed to your custom interface:

```ts
import { useTheme as useEvergreenTheme } from 'evergreen-ui';
import { Theme } from '../../components/theme';

const useTheme = (): Theme => useEvergreenTheme<Theme>();

export { useTheme };
```

Reply:
also a good idea :)